tests/test_extension.py

```diff
@@ -38,6 +38,41 @@ def test_hs_ext_binary_exporter_py3(hs_ext):
     assert not getattr(hs_ext.exporter, 'binary')
 
 
+@pytest.mark.skipif(sys.version_info < (3, 7), reason="requires python3.7")
+def test_hs_ext_dataclass_item_scraped(hs_ext):
+    from dataclasses import dataclass
+
+    @dataclass
+    class DataclassItem:
+        pass
+
+    hs_ext._write_item = mock.Mock()
+    item = DataclassItem()
+    spider = Spider('test')
+    hs_ext.item_scraped(item, spider)
+    assert hs_ext._write_item.call_count == 1
+    assert hs_ext._write_item.call_args[0] == ({'_type': 'DataclassItem'},)
+
+
+def test_hs_ext_attrs_item_scraped(hs_ext):
+    try:
+        import attr
+    except ImportError:
+        pytest.skip('attrs not installed')
+        return
+
+    @attr.s
+    class AttrsItem:
```

Comment:
```suggestion
    class AttrsItem(object):
```

Should solve [`TypeError: attrs only works with new-style classes`](https://travis-ci.org/github/scrapinghub/scrapinghub-entrypoint-scrapy/jobs/750885073#L326)

Reply:
Thanks @elacuesta